projects/ng-event-plugins/src/constants/plugins.ts

```diff
@@ -21,6 +22,7 @@ const PLUGINS = [
     SelfEventPlugin,
     StopEventPlugin,
     LongtapEventPlugin,
+    DebounceEventPlugin,
```

Comment:
this is included in the bundle by default

what if:

```ts
provideEventPlugins(DebounceEventPlugin);
```
https://github.com/taiga-family/ng-event-plugins/blob/main/projects/ng-event-plugins/src/constants/plugins.ts#L35C17-L35C36

```ts
export function provideEventPlugins(...with: Provider[]): Provider[] {
    return [...NG_EVENT_PLUGINS, ...(with ?? [])];
}
```

Reply:
I'm not sure that all plugins are needed by users and they use them, but at the same time they end up in the bundle

Reply:
I think it makes sense to have them all bundled together, they are pretty tiny. And if we want to introduce helpers for adding each of them separately – that may cause unexpected issues when some code relies on a particular plugin but it's missing.